### DISCUSSION TITLE
Capture youtube chapters automatically

### DISCUSSION THREAD

**Discussion by sammylvp** [aidenlx/media-extended#653] [Ideas]
Hi, 

I am wondering if media-extended could **automatically capture the chapters with timestamps in a youtube video**. For example [this video](https://www.youtube.com/watch?v=Abfyf-IaI4g&ab_channel=CCP4) already summarize the chapter (to see the chapters, **you need to click the "more..." link in the video introduction session right underneath the "subscribe" icon**). It would be very nice if this can be automatically included when we insert it to obsidian. 

Many thanks!
Sam

**Comment by BarRozenman** [comment]
yeah that will be a Greate feature!

**Comment by ChaplainTIG** [comment]
WOW yes this would be EPIC! Great idea @sammylvp
